src/components/ScrollView.js

```diff
@@ -331,6 +331,10 @@ const ScrollView = createReactClass({
       },
     });
   },
+  
```

Comment:
There is extra white space on this line that is failing the linter.  I'll remove it and merge this in another PR.